docs/_source/reference/python/python_annotation_metrics.rst

```diff
@@ -0,0 +1,59 @@
+.. _python_annotation_metrics:
+
+Annotation metrics
+==================
+
+Here we describe the available metrics in Argilla:
+
+- :ref:`python ref agreement_metrics`: Metrics of agreement on an annotation task
+- :ref:`python ref annotator_metrics`: Metrics for annotators. Includes both metrics per annotator and unified metrics
+  for all annotators.
+
+.. _python ref metrics:
+
+Base Metric
+-----------
+
+.. automodule:: argilla.client.feedback.metrics.base
+   :members: AgreementMetricResult, AnnotatorMetricResult
+
+.. autoclass:: argilla.client.feedback.metrics.base.MetricBase
+   :members: __init__, compute, allowed_metrics
+
+.. _python ref agreement_metrics:
+
+Agreement Metrics
+-----------------
+
+.. automodule:: argilla.client.feedback.metrics.agreement_metrics
+   :members:
+   :exclude-members: kendall_tau_dist, prepare_dataset_for_annotation_task, AgreementMetric
+
+.. autoclass:: argilla.client.feedback.metrics.agreement_metrics.AgreementMetric
+   :members: __init__, compute
+
+.. _python ref annotator_metrics:
+
+Annotator Metrics
+-----------------
+
+.. automodule:: argilla.client.feedback.metrics.annotator_metrics
+   :members:
+
+.. autoclass:: argilla.client.feedback.metrics.annotator_metrics.AnnotatorMetric
+   :members: __init__, compute
+
+.. autoclass:: argilla.client.feedback.metrics.annotator_metrics.ResponsesMetric
```

Comment:
maybe we can also unify the usage of plural vs singular here? In some cases we use plural and in others singular. 
`Annotator` vs `Suggestions`

Reply:
Sure